### PR TITLE
fix(web): don't use 2 rows for upload/download speed info

### LINKED ISSUE #6375: upload/download speed uses 2 rows
### What is the issue?

image

Once again there's something off with the UI elements. The up and download speed indicators wrap and are displayed in 2 rows. 

The previous fix doesn't work anymore:

```
sed -i 's/speed-up-label{text-align:right;width:70px}/speed-up-label{text-align:right;width:100px;white-space:nowrap}/g'
```

Update: Sorry, the fix still works. It is now in a different file, which I missed.

### Which application of Transmission?

Web client

### Which version of Transmission?

4.0.5

### PR DESCRIPTION
**Pull request by tessus**
created PR as asked by @tearfur 

fixes #6375 

Screenshot:

image

### CODE REVIEW

File: web/assets/css/transmission-app.scss [hunk 1]
```diff
@@ -322,7 +322,8 @@ $speed-icon-size: 20px;
 #speed-dn-label,
 #speed-up-label {
   text-align: right;
-  width: 70px;
+  width: 100px;
```
Comment (tearfur) [inline web/assets/css/transmission-app.scss:325]: Maybe the `width` property should be removed. The speed text and icon will move around as the speed changes, but I think this looks better than the big blank space.

See what you think.

#### Before

image

#### After

image

**Review by tearfur — COMMENTED** [review]
Thanks for the PR!

Please include screenshots for future UI PRs so that it is easier for us to review. I have uploaded them for you this time.

File: web/assets/css/transmission-app.scss (hunk 1 above)
Comment (tessus) [reply web/assets/css/transmission-app.scss:325]: Nope, I think I'd rather have a static width. If the text always moves around it is less legible and it takes for your brain more time to process the info.

Comment (tearfur) [reply web/assets/css/transmission-app.scss:325]: Then maybe make it narrower?

The biggest unit possible here is MB/s. Having over 1000 MB/s transfer speed is not impossible, but quite a far fetch, so I think we can assume `xxxx.xx MB/s` is the widest string possible.

Comment (tessus) [reply web/assets/css/transmission-app.scss:325]: The entire idea of this PR was to not wrap and use 2 rows. I've tested 100px and I never ran into any issue ever again. 70 vs 100 may seem a huge increase percentage wise, but what are 100px these days?
Are you running the web interface on a 300px device?
I haven't used a display less than 1000px for more than 10 years. Even my Pixel 5 has 1080 x 2340 pixels.

Btw, `xxxx.xx MB/s` is actually wider than 100px.

**Review by tearfur — APPROVED** [review]

**Comment by tessus** [comment]
I thought it was fixed in the main branch in some other way. Wouldn't a PR for the main branch be useless then? Although I actually don't see any changes in this regard. Maybe it wasn't fixed after all. I don't have an issue creating another PR for the main branch. Just wondering...

**Comment by tessus** [comment]
done #6577

**Comment by tearfur** [comment]
> I thought it was fixed in the main branch in some other way. Wouldn't a PR for the main branch be useless then?

That's correct AFAIK, not sure what @Coeur meant there.

**Comment by Coeur** [comment]
Ah, good. Just wanted to make sure no fix was lost.

**Comment by tearfur** [comment]
> But I am afraid this will move the upload/download speed all the time. Personally I am not a fan of this.

I test my dev builds regularly with the WebUI in `main`, and this isn't bothering me at all.

Let's keep it as-is for now. If more people complain about it, then we'll figure out something for it.

**Comment by tessus** [comment]
> I test my dev builds regularly with the WebUI in `main`, and this isn't bothering me at all.

This is strange. If I am looking at something and the text shifts left and right all the time, it makes me nervous and the information it wants to convey is harder to parse.
How does this not bother you? I am actually amazed and wished my brain worked differently in that sense.

Anyway, is something still missing for this to be merged?

**Comment by tearfur** [comment]
> Anyway, is something still missing for this to be merged?

Good to go in my book. Just waiting for someone with write access to look at this PR.

**Comment by Coeur** [comment]
> Good to go in my book. Just waiting for someone with write access to look at this PR.

@ckerr can we grant Tearfur write access? And I would maybe apply for membership to the project, if that can help handling the macOS pull requests of the project.

**Comment by tearfur** [comment]
That'll be convenient for me for sure. But if there's anything expected from me in exchange of having write access, please let me know beforehand.

**Comment by GaryElshaw** [comment]
I think the survival of the project requires another person to have write access.

**Comment by Coeur** [comment]
Congratulations on your upgrade, Tearfur. :)

**Comment by tessus** [comment]
so.... are we waiting for 4.0.6 and mege it then or what's the hold up? Just asking because I created the PR because I was asked for it weeks ago.

**Comment by tearfur** [comment]
> so.... are we waiting for 4.0.6 and merge it then or what's the hold up?

@ckerr

**Comment by tearfur** [comment]
@ckerr I'd also like this PR to be included in `4.0.6`.

**Comment by ckerr** [comment]
Notes: fixed layout issue in speed display.

**Pull request merged**